internal/template/consts_on_types.md.gotmpl

```diff
@@ -0,0 +1,15 @@
+{{ define "constsTypes" }}
```

Comment:
nit: maybe rename file to `types_consts.md.gotmpl`, and template name to `"typesConsts"`? (same for vars).

Reply:
Fixed!